react/features/e2ee/middleware.js

```diff
@@ -191,6 +191,29 @@ MiddlewareRegistry.register(({ dispatch, getState }) => next => action => {
 
         break;
     }
+
+    case SET_MEDIA_ENCRYPTION_KEY: {
+        if (conference && conference.isE2EEEnabled()) {
```

Comment:
We need to allow for "forgetting" a key. That is, setting it to false / undefined, even when encryption is not on. Actually, we should allow setting the keys while e2ee is not on because they will want to set them first, and start using them later.